src/asmcnc/apps/drywall_cutter_app/material_setup_popup.py

```diff
@@ -326,13 +325,18 @@ def __init__(self, localization, keyboard, dwt_config, **kwargs):
 
     def on_open(self):
         self.load_active_config()
+        self.update_text()
 
     def get_safe_float(self, val):
         try:
             return float(val)
         except:
             return 0.0
 
+    def text_on_focus(self, instance, value):
+        if not value:
```

Comment:
I might be understanding this wrong, but wouldn't this only validate when the textinput is empty?